filament/backend/src/webgpu/WebGPUConstants.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef TNT_FILAMENT_BACKEND_WEBGPUCONSTANTS_H
+#define TNT_FILAMENT_BACKEND_WEBGPUCONSTANTS_H
+
+#include <cstdint>
+
+#include "utils/Log.h"
+
+// FWGPU is short for Filament WebGPU
+
+// turn on runtime validation, namely for debugging, that would normally not run (for release)
+#define FWGPU_DEBUG_VALIDATION     0x00000001
```

Comment:
A lot of this seems based on the vulkan backend it seems. Not a critique, but just kind of a FYI --> 

Each of the backend has, historically, been owned by a single individual, and each of them has its own style because of that. I put together these constant things because I kind of preferred it this way.  But no one says that the webgpu should be designed/implemented this way.  In short, you have some freedom here on how you'd like to implement the backend. You can also look at metal and gl for some inspirations.